FutureNorthants/northants-website
# Feature/video embed

Add video slice to embed a video from YouTube or Vimeo in the backend. 

## To Test

* Checkout branch
* Import config - `drush config:import`
* Edit a service page and add a video slice, then save and publish the page
* Use the following query in the graph ql tester (entering the path of the service page) and ensure the correct `video_id`, `provider` and `description` are returned: 

```
query {
  route(path: "") {
    __typename
    ... on DrupalNodeRoute {
      node {
        ... on ServicePageNode {
          __typename
          title
          id
          serviceBody: body {
            value
            embeds {
              id
              paragraph {
                __typename
                ... on VideoParagraph {
                  video_id
                  provider
                  description
                  id
                }
              }
              __typename
            }
            __typename
          }
        }
      }
    }
  }
}
```
